README.md

```diff
@@ -0,0 +1,34 @@
+# XMTP React playground
+
+Use the XMTP React playground app as a tool to help you build your own app with XMTP.
+
+Built with React and the [XMTP JavaScript SDK](https://github.com/xmtp/xmtp-js), the playground provides the following functionality that you can use as a foundation for your app:
+
+- Performant [database architecture](https://xmtp.org/docs/tutorials/performance#use-a-local-cache)
+- Group chat
+- Message reactions
+- Message replies
+- [Attachments](https://xmtp.org/docs/build/attachments)
+  - Images
+  - More formats to come
+
+The playground's visual design is intentionally limited to help make the code easier for your to use, customize, and deploy.
+
+The playground has not undergone a formal security audit.
+
+> **Note**  
+> You might also be interested in our Vue playground: Coming soon 
+
+## Install the package
+
+```bash
+npm install
+```
+
+## Run the development server
+
+```bash
+npm run dev
+```
+
+Open [http://localhost:5173](http://localhost:5173) in your browser to access the XMTP React playground app.
```

Comment:
@nplasterer Just an FYI that I updated the port number from 3000 to 5173 based on what opened when I ran this command. Do we want to update the code to make it open on port 3000? <3

Reply:
This is the port it opened for me too so this seems fine!